jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/ErrorHandlerTest.java

```diff
@@ -159,6 +160,30 @@ public void test404NoAccept() throws Exception
         assertContent(response);
     }
 
+    @Test
+    public void test404NoAcceptButSpecifiedDefaultResponseMimeType() throws Exception
+    {
+        ErrorHandler errorHandler = new ErrorHandler();
+        errorHandler.setDefaultResponseMimeType(MimeTypes.Type.APPLICATION_JSON);
+        server.setErrorHandler(errorHandler);
+
+        String rawResponse = connector.getResponse(
+                "GET / HTTP/1.1\r\n" +
+                        "Host: Localhost\r\n" +
+                        "\r\n");
```

Comment:
Use a text block here 😃

Reply:
Haven't used Java in years ;-) Nice that that is possible. I updated the test.